src/phone-input/__tests__/phone-input.scenario.js

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+
+import {StatefulPhoneInput} from '../index.js';
+import {Block} from '../../block/index.js';
+
+export const name = 'phone-input';
+
+export const component = () => (
+  <Block>
+    <StatefulPhoneInput aria-label="stateful phone input example" />
```

Comment:
Is there a default `aria-label` provided?

Reply:
yeah, it is `Please choose a country dial code and enter a phone number.`